### ISSUE TITLE
[Question] Provider support via support.jfrog.com?

### ISSUE THREAD

**Issue by skyzyx** [jfrog/terraform-provider-artifactory#171]
Breaking out from #167 —

Can I file a ticket at https://support.jfrog.com if I need an expedited response to an issue in this Terraform Provider?

Some companies have explicitly stated that they will absolutely not support OSS projects via paid support channels. Other companies include it because their OSS offerings are part of their holistic developer experience. Wondering where JFrog leans on this issue.

**Comment by chb0github** [comment]
It's like this:

The terraform provider is not a core jfrog product and as such doesn't have its own dedicated support channel. Internally, it's tracked as part as our hashicorp integration efforts (`Partner/ACC-007450`). So, what does that mean:

I am quite literally the only one watching issues through GH and (currently), I have no other priorities, but that won't last forever. So, submitting a ticket through support won't get you any extra momentum ('cause there is no one else to help). However, it will get your issue noticed and rolled up in to actual product metrics. If priorities should change, and traction is lost through GH, then it's definitely your best bet.

Bottom line: If I say `submit a support ticket`, read between the lines and do so. If tickets on GH start getting ignored: Issue a support ticket; likely I am dead or at least dead to jfrog 🤣
